### DISCUSSION TITLE
Windows Binaries for Geany Plug In's

### DISCUSSION THREAD

**Discussion by RoyiAvital** [geany/geany#2786] [General]
I'm after a portable installation of Geany (Based on [Geany from PortableApps](https://portableapps.com/apps/development/geany_portable)).

I need the Plug In's in their final (Binary?) form to put in the portable folder.
Where can I find them and not through Chocolatey?

**Comment by elextr** [comment]
The portable apps are not made by the Geany project, whilst someone here may know, there are possibly better places to ask like on https://portableapps.com/

**Reply by RoyiAvital** [reply]
I am just after the DLL's of the Plug In's.

Once I have them I know what to do.
I was just mentioning PortableApps in case others want a portable Geany.

**Reply by elextr** [reply]
There is a windows installer for the plugin collection on https://www.geany.org/ downloads page.

**Reply by RoyiAvital** [reply]
Does `geany-plugins-x.xxx_setup.exe` mean Geany + Selected Plug In's? Or is it only Plug In's?

**Reply by elextr** [reply]
Only OSX combines plugins and Geany, for Windows Geany is standalone and plugins can be added.
